### PR TITLE
Feat/6 changed service alert feed timestamp

### PR DESCRIPTION
**Pull request by Divinare**
Closes https://github.com/HSLdevcom/transitdata/issues/6

### CODE REVIEW

File: src/test/java/fi/hsl/transitdata/omm/models/AlertStateTest.java [hunk 1]
```diff
@@ -209,9 +209,8 @@ private AlertState createModifiedAlertState(final List<Bulletin> secondBulletins
     public void testLastModified() throws Exception {
         List<Bulletin> bulletins = readDefaultTestBulletins();
         AlertState state = new AlertState(bulletins);
-        Optional<LocalDateTime> maybeDt = state.lastModified();
-        assertTrue(maybeDt.isPresent());
-        String dt = DAOImplBase.OMM_DT_FORMATTER.format(maybeDt.get());
+        LocalDateTime lastModified = state.lastModified();
+        String dt = DAOImplBase.OMM_DT_FORMATTER.format(lastModified);
         assertEquals("2019-05-08 09:44:54", dt);
```
Comment (kenkus-futurice) [inline src/test/java/fi/hsl/transitdata/omm/models/AlertStateTest.java]: This and another test is failing:
https://travis-ci.org/HSLdevcom/transitdata-omm-alert-source

**Review by kenkus-futurice — CHANGES_REQUESTED** [review]
Tests are failing, otherwise LGTM

**Comment by Divinare** [comment]
Broken tests are removed now because they were not needed. Also refactored code a bit

**Review by kenkus-futurice — APPROVED** [review]
LGTM

**Pull request merged**